query-algebrizer/tests/utils/mod.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright 2018 Mozilla
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+use mentat_core::{
+    Attribute,
+    Entid,
+    Schema,
+    ValueType,
+};
+
+use mentat_query_parser::{
+    parse_find_string,
+};
+
+use mentat_query::{
+    NamespacedKeyword,
+};
+
+use mentat_query_algebrizer::{
+    algebrize,
+    algebrize_with_inputs,
+    ConjoiningClauses,
+    Error,
+    QueryInputs,
+};
+
+// Common utility functions used in multiple test files. Note: Import this with
+// `pub mod utils` (not `mod utils`), or you'll get spurious unused function
+// warnings when functions exist in this file but are only used by modules that
+// don't import with `pub` (yes, this is annoying).
```

Comment:
Better to add `#![allow(unused_code)]` underneath the block comment in this file, and not use `pub mod`.